tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenAbortTransactionTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿/* Copyright 2018-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System.Collections.Generic;
+using System.Threading;
+using MongoDB.Bson;
+using MongoDB.Bson.TestHelpers.JsonDrivenTests;
+
+namespace MongoDB.Driver.Tests.JsonDrivenTests
+{
+    public sealed class JsonDrivenAbortTransactionTest : JsonDrivenClientTest
```

Comment:
All the new JSON driven tests are in this directory.

Eventually (soon I hope) we should refactor the CRUD tests to use these also.